.github/workflows/Construct.yml

```diff
@@ -0,0 +1,49 @@
+name: Build installers
+
+on:
+  workflow_call:
+
+permissions:
+  contents: write    
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  openlane-sky130a-installer:
+    runs-on: "ubuntu-20.04"
+    steps:
+      - name: checkout conda-eda
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # fetch all tags
+
+      - name: construct installer
+        run: |
+          export CI_SCRIPTS_PATH="$(pwd)/ci"
+          ./ci/install.sh
+          conda install constructor
+          constructor installer/openlane.sky130a/
+
+      - name: archive openlane.sky130a installer
+        uses: actions/upload-artifact@v3
+        with:
+          name: "openlane-sky130a-installer"
+          path: |
+            /home/runner/work/conda-eda/conda-eda/openlane-sky130a-0-Linux-x86_64.sh
```

Comment:
Isn't there any variable available with this path?

Reply:
I think that would be `${{ env.GITHUB_WORKSPACE }}/openlane-sky130a-0-Linux-x86_64.sh`

Reply:
Using `openlane-sky130a-0-Linux-x86_64.sh` should also work. Since `actions/checkout@v3` is used, the default location for other steps is the root of the repo already.

Reply:
FWIW I have no preference on the two.